core/sched.c

```diff
@@ -221,3 +202,36 @@ NORETURN void sched_task_exit(void)
     sched_active_thread = NULL;
     cpu_switch_context_exit();
 }
+
+#ifdef MODULE_SCHED_CB
```

Comment:
I would namespace this one with `CORE_` but it would break the submodules blind handling.

Reply:
It would be nice to specify core related modules, would you want it handle in this PR or a follow-up?

Reply:
Better have this that waiting on my PR I would say.

If you do not want to add a module that will be renamed, you could also keep it in `#ifdef MODULE_SCHEDSTATISTICS` as it was before.